include/k4a/k4atypes.h

```diff
@@ -280,15 +280,13 @@ typedef enum
  * \remarks
  * Binned modes reduce the captured camera resolution by combining adjacent sensor pixels into a bin.
  *
- * \remarks
- * If modes are added, update the k4a_depth_mode_strings array.
- *
  * \xmlonly
  * <requirements>
  *   <requirement name="Header">k4atypes.h (include k4a/k4a.h)</requirement>
  * </requirements>
  * \endxmlonly
  */
+// Be sure to update k4a_depth_mode_to_string in k4a.c if enum values are added.
```

Comment:
I did have that initially before pushing my changes but Brent recommended pulling this comment out of the doxygen block entirely. Suggestion @Brent-A ?

Reply:
/internal is mainly used if we want to produce a second set of doxygen output explaining how to use internal components. For this sort of comment, which isn't about the usage of the API but more about how to make edits to the file, I'm not sure that it needs to go in any doxygen docs. Since we aren't building docs with /internal, it doesn't really matter.